<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1559</width>
    <height>1164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1500</width>
      <height>1000</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>1040</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>1040</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connection settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testconnButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>1040</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>1000</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SCORE</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="p1Score">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>1050</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>1040</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="p2Score">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>1050</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="connStatus">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>1020</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="startSaving">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>1040</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Saving</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replayButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>1040</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1559</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
